### PR TITLE
fundamental-react: add properties to SideNav and MenuItem 

### PR DESCRIPTION
**Pull request by ThomHuynh**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test YOUR_PACKAGE_NAME`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://sap.github.io/fundamental-react/?path=/docs/component-api-sidenav--primary#properties , https://sap.github.io/fundamental-react/?path=/docs/component-api-sidenav--primary#properties
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

File: types/fundamental-react/lib/Menu/Menu.d.ts [hunk 1]
```diff
@@ -14,12 +14,16 @@ export type MenuGroupProps = {
...
-    addon?: string;
+    active?: boolean;
+    addonAfter?: string;
     addonBefore?: string;
     addonProps?: any;
     className?: string;
+    disabled?: boolean;
     isLink?: boolean;
-    onclick?: (...args: any[]) => any;
+    onClick?: (...args: any[]) => any;
```
Comment (auryn31) [inline types/fundamental-react/lib/Menu/Menu.d.ts:24]: Please add the correct type here

File: types/fundamental-react/lib/SideNavigation/SideNav.d.ts [hunk 2]
```diff
@@ -25,7 +27,9 @@ export type SideNavListProps = {
 } & React.HTMLAttributes<Element>;
 
 export type SideNavListItemProps = {
+    condensed?: boolean;
```
Comment (auryn31) [inline types/fundamental-react/lib/SideNavigation/SideNav.d.ts:30]: did you have a link for this change?

**Review by auryn31 — CHANGES_REQUESTED** [review]

File: types/fundamental-react/lib/SideNavigation/SideNav.d.ts (hunk 2 above)
Comment (ThomHuynh) [reply types/fundamental-react/lib/SideNavigation/SideNav.d.ts:30]: link added to my initial comment 👍

File: types/fundamental-react/lib/Menu/Menu.d.ts (hunk 1 above)
Comment (ThomHuynh) [reply types/fundamental-react/lib/Menu/Menu.d.ts]: changed it to `    onClick?: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
`

**Review by auryn31 — APPROVED** [review]
LGTM 👍

**Comment by ThomHuynh** [comment]
Ready to merge

**Pull request merged**